plugins/README.md

```diff
@@ -54,12 +54,13 @@ C++ builds may require a specific toolchain: `--config=clang` or `--config=gcc`.
 
 # Testing and benchmarking
 
-1.  Write a plugin test file (text proto) to specify the plugin's functional
-    expectations ([example](samples/testing/tests.textpb)). Consult the plugin
-    tester [proto API](test/runner.proto) as needed.
-1.  Add `benchmark: true` to tests that exemplify common wasm operations
+1. Write a plugin test file (text proto) to specify the plugin's functional
+    expectations ([example](samples/testing/tests.textpb)). 
+2. YAML test file is also supported ([example](samples/testing/tests.yaml)).
```

Comment:
Suggest folding items 2 and 3 into item 1 in the list, since they are not separate steps, but rather options and considerations under 1. Proposed wording:

1. Write a plugin test file to specify the plugin's functional expectations. This file can either be in text proto format ([example](samples/testing/tests.textpb)) or YAML ([example](samples/testing/tests.yaml)). Consult the plugin tester [proto API](test/runner.proto) as needed.